primitives/src/constants.rs

```diff
@@ -50,3 +50,35 @@ pub mod time {
 	};
 	pub const INFINITY: u32 = u32::MAX;
 }
+
+pub mod chain {
+	pub use crate::{AssetId, Balance};
+	pub use frame_support::weights::{constants::WEIGHT_PER_SECOND, Weight};
+	pub use sp_runtime::Perbill;
+
+	/// Core asset id
+	pub const CORE_ASSET_ID: AssetId = 0;
+
+	/// Max fraction of pool to buy in single transaction
+	pub const MAX_OUT_RATIO: u128 = 3;
+
+	/// Max fraction of pool to sell in single transaction
+	pub const MAX_IN_RATIO: u128 = 3;
+
+	/// Trading limit
+	pub const MIN_TRADING_LIMIT: Balance = 1000;
+
+	pub const RUNTIME_AUTHORING_VERSION: u32 = 1;
```

Comment:
runtime variables should be probably in the runtime common instead and its cargo version in sync with spec version

Reply:
or mby it shouldn't be extracted at all

Reply:
also the spec version should be bumped to 10 already

Reply:
I have moved runtime variables to common-runtime, synced cargo and spec versions and bumped the spec version to 10.